gpflux/layers/basis_functions/fourier_features/random/base.py

```diff
@@ -74,7 +79,17 @@ def _sample_students_t(nu: float, shape: ShapeType, dtype: DType) -> TensorType:
 
 class RandomFourierFeaturesBase(FourierFeaturesBase):
     def __init__(self, kernel: gpflow.kernels.Kernel, n_components: int, **kwargs: Mapping):
-        assert isinstance(kernel, RFF_SUPPORTED_KERNELS), "Unsupported Kernel"
+        assert isinstance(kernel, (RFF_SUPPORTED_KERNELS, RFF_SUPPORTED_MULTIOUTPUTS)), (
+            "Unsupported Kernel; only SquaredExponential, Matern12, Matern32, Matern52, "
+            "or Shared/SeparateIndependent kernels using these are supported"
+        )
```

Comment:
This might be more maintainable:
```python3
if not isinstance(kernel, (RFF_SUPPORTED_KERNELS, RFF_SUPPORTED_MULTIOUTPUTS)):
   raise TypeError("Unsupported Kernel: only the following kernel types are supported: {[k.__name__ for k in RFF_SUPPORTED_MULTIOUTPUTS + RFF_SUPPORTED_KERNELS]}")
```

Reply:
I agree